src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using Dynamo.Interfaces;
+using DelegateCommand = Dynamo.UI.Commands.DelegateCommand;
+
+
+namespace Dynamo.ViewModels
+{
+    class PackagePathViewModel : ViewModelBase
+    {
+        public ObservableCollection<string> RootLocations { get; private set; }
+        private int selectedIndex;
+        public int SelectedIndex {
+            get { return selectedIndex; }
+            set 
+            {
+                selectedIndex = value;
+                if (selectedIndex < RootLocations.Count - 1)
+                    MovePathDownCommand.RaiseCanExecuteChanged();
+                if (selectedIndex > 0)
+                    MovePathUpCommand.RaiseCanExecuteChanged();
+            }
+        }
```

Comment:
Forget about the `if` conditions, just raise the events regardless. Also, the first curly bracket should be on a new line.